tests/components/lcn/test_init.py

```diff
@@ -125,3 +125,29 @@ async def test_async_setup_from_configuration_yaml(hass: HomeAssistant) -> None:
         await setup_component(hass)
 
         assert async_setup_entry.await_count == 2
+
+
+@patch("homeassistant.components.lcn.PchkConnectionManager", MockPchkConnectionManager)
+async def test_migrate_1_1(hass: HomeAssistant, entry) -> None:
+    """Test migration config entry."""
+    hass.http = Mock()
+    entry.add_to_hass(hass)
+    data = {**entry.data}
+
+    # diff 1.2 -> 1.1
+    data.pop(CONF_ACKNOWLEDGE)
+
+    hass.config_entries.async_update_entry(
+        entry,
+        version=1,
+        minor_version=1,
+        data=data,
+    )
```

Comment:
Should we just create a new MockConfigEntry just so the test will keep working in the future, independent of what happens with the `entry` fixture?

Reply:
Done